docs/packages.toml

```diff
@@ -120,6 +120,11 @@
 #   user = "GAMS-dev"
 # [Hypatia]
 #   user = "chriscoey"
+[InfiniteOpt]
+  user = "infiniteopt"
+  rev = "jump-readme"
```

Comment:
Change before merging

Reply:
I'll change it to the new tag which will be `v0.5.7`. I should have the new version tagged within an hour or so, I want to release some patch fixes anyway.

Reply:
Cool. Ping here when you're ready for a rebuild.

Reply:
Ok, `v0.5.7` is released.